app/src/main/java/com/kkolontay/theworld/ui/screens/InfoView.kt

```diff
@@ -0,0 +1,34 @@
+package com.kkolontay.theworld.ui.screens
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun InfoView(modifier: Modifier = Modifier
+    .fillMaxWidth()
+    .padding(16.dp), taps: Int, refresh: () -> Unit, back: Int) {
```

Comment:
It's recommended to place each function parameter on it's own line for readability.  Also, lamba parameters are usually placed at the end:

```kotlin
fun InfoView(
    modifier: Modifier = Modifier
        .fillMaxWidth()
        .padding(16.dp),
    taps: Int, 
    back: Int,
    refresh: () -> Unit,
) {
```

The reason for placing lambdas at the end are so that way the callsite reference can be simplified:

```kotlin
InfoView(modifier: modifier, taps: tapsVal, back: backVal) {
    // handle refresh lambda
}
```